moduleroot/CONTRIBUTING.md.erb

```diff
@@ -3,4 +3,4 @@
 This code repository is part of Project Syn and the contribution guide at
 https://syn.tools/syn/about/contribution_guide.html does apply.
 
-Submit Pull Requests at <%= @configs['githubUrl'] %>/pulls.
```

Comment:
We could also remove this variable from the :globals in config_defaults.yml if it's unused.